test/lens.spec.js

```diff
@@ -242,25 +205,24 @@ describe("Lens Functions", () => {
     })
   })
   describe("domLens", () => {
-    it("value", () => {
+    it("domLens.value", () => {
```

Comment:
No need for `domLens.` since the group is already named `domLens`

Reply:
@stellarhoof The it(string, ) has to match the title that appears in the docs. That's how they get matched. In the docs we have F.domLens.value and more, so the `it` has to be it('domLens.value', ...). Even so, sometimes the test is not found and hence there's no example for a function.

Reply:
Got it